org.messerlab.slimgui.yml

```diff
@@ -0,0 +1,43 @@
+app-id: org.messerlab.slimgui
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+  - --talk-name=com.canonical.AppMenu.Registrar # required for global menu
+  - --talk-name=org.freedesktop.Flatpak # allows spawning processes on the host via flatpak-spawn --host
```

Comment:
this is not allowed by default

An exception need to be requested

See https://docs.flathub.org/docs/for-app-authors/linter/

Reply:
@bryce-carson I guess this is one for you!  :->

Reply:
> this is not allowed by default
> 
> An exception need to be requested
> 
> See https://docs.flathub.org/docs/for-app-authors/linter/

How do I request an exception?

Eidos, the SDL which SLiMgui uses for its functionality, has a `system()` function which must be workable to create other processes, such as an R process, or anything the user might otherwise want to call.

Reply:
> > See https://docs.flathub.org/docs/for-app-authors/linter/
> 
> How do I request an exception?

You didn't read the documentation linked above?